conda_libmamba_solver/solver.py

```diff
@@ -453,6 +453,10 @@ def _specs_to_tasks_add(self, in_state: SolverInputState, out_state: SolverOutpu
                             "UPDATE | ESSENTIAL | FORCEBEST",
                             api.SOLVER_UPDATE | api.SOLVER_ESSENTIAL | api.SOLVER_FORCEBEST,
                         )
+                    elif self._command == "update":
```

Comment:
Hm, we already checked for this in line 445 above. I wonder if we can simply change the `!=` in the else below? I remember experimenting with this but something else broke. And we also need to check for the error messages. If a package cannot be updated because the latest version is already there, then that's not an error per se.